app/src/main/java/io/apicurio/registry/rules/RulesServiceImpl.java

```diff
@@ -61,17 +66,26 @@ public void applyRules(String artifactId, ArtifactType artifactType, ContentHand
             rules = storage.getGlobalRules();
             useGlobalRules = true;
         }
-        if (rules.isEmpty()) {
-            return;
+
+        List<RuleType> defaultGlobalRules = rulesProperties.getDefaultGlobalRules(rules);
+        if (!defaultGlobalRules.isEmpty()) {
+            // We have default global rules as well as rules from storage, so use the concatenated list
+            rules = Stream.concat(rules.stream(), defaultGlobalRules.stream())
+                .collect(Collectors.toList());
         }
+
         ContentHandle currentArtifactContent = null;
         if (ruleApplicationType == RuleApplicationType.UPDATE) {
             StoredArtifact currentArtifact = storage.getArtifact(artifactId);
             currentArtifactContent = currentArtifact.getContent();
         }
         for (RuleType ruleType : rules) {
-            RuleConfigurationDto configurationDto = useGlobalRules ?
-                                                    storage.getGlobalRule(ruleType) : storage.getArtifactRule(artifactId, ruleType);
+            RuleConfigurationDto configurationDto;
+            if (defaultGlobalRules.contains(ruleType)) {
+                configurationDto = rulesProperties.getDefaultGlobalRuleConfiguration(ruleType);
+            } else {

```

Comment:
Do we want the **default** rules to take precedent over configured rules?  Probably not?

Reply:
As `defaultGlobalRules` only contains rules that are not found in the rules from storage, then we know to retrieve the rule configuration from the default global rules here. Otherwise the rule must be from storage, so we get the configuration from storage.